_posts/2024-04-08-WildFly32Beta-released.adoc

```diff
@@ -11,23 +11,24 @@ description: WildFly 32 Beta is now available for download.
 
 I'm pleased to announce that the new WildFly and WildFly Preview 32.0.0.Beta1 releases are available for download at https://wildfly.org/downloads.
 
-This is our usual single beta that we release a couple weeks before the 32 Final release. We typically don't do much of a release announcement post for the beta release, but there are a number of things happening with WildFly 32 that users of our betas should know about and that we encourage people who usually don't look at our betas to try out.
+This is our usual single beta that we release a couple of weeks before the 32 Final release. We typically don't do much of a release announcement post for the beta release, but there are a number of things happening with WildFly 32 that users of our betas should know about and that we encourage people who usually don't look at our betas to try out.
 
 == Feature Stability Levels
 
 There a number of new features in WildFly 32 Beta1. I'll wait for the Final release announcement to describe them all; for now I'll just direct you to the link:https://github.com/wildfly/wildfly/releases/tag/32.0.0.Beta1[release notes].
 
 Something that's pretty new in 32 Beta though is that we've started to take advantage of the capabilities introduced in WildFly 31 to introduce link:https://docs.wildfly.org/31/Admin_Guide.html#Feature_stability_levels[features at different stability levels]. The goal here is to allow users who want to look at features in earlier stages of the development lifecycle to easily do so, without leaving users who are not interested in that in a situation where they may inadvertently use those features.
 
-As noted in link:https://docs.wildfly.org/31/Admin_Guide.html#Feature_stability_levels[the documentation], new features are offered at one of four stability levels: `experimental`, `preview`, `commmunity` and 'default`. Most features in both standard WildFly and WildFly Preview are at `default` stability, but increasing numbers of new features will be introduced at the other levels, and hopefully will be promoted in later releases up to `community` or `default`.
+As noted in link:https://docs.wildfly.org/31/Admin_Guide.html#Feature_stability_levels[the documentation], new features are offered at one of four stability levels: `experimental`, `preview`, `community` and 'default`. Most features in both standard WildFly and WildFly Preview are at `default` stability, but increasing numbers of new features will be introduced at the other levels, and hopefully will be promoted in later releases up to `community` or `default`.
```

Comment:
@rhusar can you fix the ticks around default in your PR?

Reply:
@darranl Nice, fixed that too.